SAMLPortal/wwwroot/js/site.js

```diff
@@ -2,3 +2,47 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+function getResultsFromFilters()
+{
+	let uri = "/api/setup/getResultsFromFilters";
+	let results = [];
+	let testUsername = document.getElementById("testUsername").value;
+
+	fetch(uri, {
+		method: "POST",
+		headers: {
+			'Content-Type' : 'application/json'
+		},
+		body: JSON.stringify(testUsername)
+	})
+	.then(response => response.json())
+	.then((response) => {
+		console.log(response);
+		let values = response['value']
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)